app/helpers/date_time_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module DateTimeHelper
+  def format_time(datetime)
+    datetime.strftime("%-l:%M%P")
+  end
+
+  def scheduled_date_placeholder
```

Comment:
I think if this had a more generic name, e.g. `date_time_placeholder` this helper and the `TimeOptionsHelper` could be merged.